project/src/main/ui/settings/keybind/settings-keybinds-control.gd

```diff
@@ -165,22 +167,36 @@ func _assign_wasd_joypad_values() -> void:
 	_preset_control("SwapHoldPiece").visible = SystemData.gameplay_settings.hold_piece
 
 
+func _refresh_focus_neighbours() -> void:
```

Comment:
Comment

Reply:
Added comment: `Assigns the TabContainer's bottom focus neighbor to the pressed preset button.`